pyOCD/utility/conversion.py

```diff
@@ -37,6 +38,20 @@ def word2byte(data):
         res.append((x >> 24) & 0xff)
     return res
 
+## @brief Convert a halfword array into a byte array
+def half2byte(data):
+    byteData = []
+    for i in data:
+        byteData.extend([data[i] & 0xff, (data[i] >> 8) & 0xff])
+    return byteData
+
+## @brief Convert a byte array into a halfword array.
+def byte2half(byteData):
+    data = []
+    while i in range(0, len(byteData), 2):
```

Comment:
Oh, the original code was correct (though not the function names!), it just had a funny name for the **for** loop variable.